dags/hivemind_etl_helpers/tests/integration/test_get_gdrive_community_data.py

```diff
@@ -0,0 +1,271 @@
+from datetime import datetime
+from unittest import TestCase
+
+from bson import ObjectId
+from hivemind_etl_helpers.src.utils.get_communities_data import (
+    get_google_drive_communities,
+)
+from hivemind_etl_helpers.src.utils.mongo import MongoSingleton
+
+
+class TestGetDriveCommunityData(TestCase):
+    def setUp(self):
+        client = MongoSingleton.get_instance().client
+        client["Core"].drop_collection("modules")
+        client["Core"].drop_collection("platforms")
+
+        self.client = client
+
+    def test_get_empty_data(self):
+        result = get_google_drive_communities()
+        self.assertEqual(result, [])
+
+    def test_get_single_data(self):
+        self.client["Core"]["modules"].insert_one(
+            {
+                "name": "hivemind",
+                "communityId": ObjectId("6579c364f1120850414e0dc5"),
+                "options": {
+                    "platforms": [
+                        {
+                            "platformId": ObjectId("6579c364f1120850414e0dc6"),
+                            "type": "source",
+                            "fromDate": datetime(2024, 1, 1),
+                            "options": {},
+                        }
+                    ]
+                },
+            }
+        )
+        self.client["Core"]["platforms"].insert_one(
+            {
+                "_id": ObjectId("6579c364f1120850414e0dc6"),
+                "name": "google-drive",
+                "metadata": {
+                    "name": "TEST",
+                    "folder_ids": [
+                        "1RFhr3-KmOZCR5rtp4dlOM",
+                        "1RFhr3-KmOZCR5rtp4dlON",
+                        "1RFhr3-KmOZCR5rtp4dlOO",
+                    ],
+                    "file_ids": [
+                        "1RFhr3-KmOZCR5rtp12",
+                        "1RFhr3-KmOZCR5rtp13",
+                        "1RFhr3-KmOZCR5rtp14",
+                    ],
+                    "drive_ids": "1RFhr3-KmOZCR5rtp123",
```

Comment:
`drive_ids` should hold a list of strings. Please consider updating the test case